samples/powerpoint/tags/tags.yaml

```
@@ -153,21 +197,29 @@ template:
     content: |-
         <section>
             <p>These snippets show how to use tags with the presentation and its slides and shapes.</p>
+        </section>
+        <section>
+            <h3>Set up</h3>
+            <p>Add several slides to the deck. Add content to each so they are visually distinct in the thumbnail pane.</p>
+            <button id="setup">Set up</button>
+        </section>
         <section>
             <h3>Try it out</h3>
-            <p>1. Add several slides to the deck. Add content to each so they are visually distinct in the thumbnail pane.</p>
-            <p>2. Select a <i>single</i> slide and press <b>Add tag</b> to tag the slide to be shown only to premium customers.</p>
+            <p>1. Select a <i>single</i> slide and press <b>Add tag</b> to tag the slide to be shown only to premium customers.
+            </p>
             <button id="add-selected-slide-tag">Add tag</button>
-            <p>3. Repeat step 2 for another slide.</p>
-            <p>4. Press <b>Delete premium customer slides</b> to remove from the presentation slides that should only be shown to premium customers.</p>
+            <p>2. Repeat step 2 for another slide.</p>
```

Should be step 1 now?
```suggestion
            <p>2. Repeat step 1 for another slide.</p>
```